src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div id="content" class="app-notestutorial">
+	<div id="vue-content" class="app-notestutorial">
```

Comment:
I think that should use the AppContent component and then the problem is gone?!

Reply:
You're right we are already using `NcAppContent` - so we have a main tag on the page.